docker/datahub-mae-consumer/debug/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Dummy docker file for development. Assumes code is built on your machine and mounted to this image.
```

Comment:
I assume these dummy dockerfiles will be more of less the same or maybe create a base image instead of repeating code?

Reply:
They are pretty much the same, yes, but idk there's anyway to reuse what's in these files... we'd probably have to publish the "base" images to dockerhub... eh?

Reply:
Does this help? https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Yes and no? It's neat, but I don't think there's a way to use it with compose? Seems like `docker build` only :(

Reply:
ah wait there is 

https://docs.docker.com/compose/compose-file/#target

Reply:
done